src/modules/workspaces/attributes/workspace-logs.attributes.ts

```diff
@@ -0,0 +1,34 @@
+export enum WorkspaceLogType {
+  LOGIN = 'LOGIN',
```

Comment:
The [standard](https://www.typescriptlang.org/docs/handbook/enums.html) is to use camelCase for the enum keys and you can also use lowercase for the values like: 

```
export enum WorkspaceLogType {
  LogIn = 'login',
  PasswordUpdated = 'password-updated',
  ...
}
```